task-sdk/src/airflow/sdk/execution_time/context.py

```diff
@@ -272,7 +273,12 @@ class ConnectionAccessor:
     """Wrapper to access Connection entries in template."""
 
     def __getattr__(self, conn_id: str) -> Any:
-        return _get_connection(conn_id)
+        try:
+            return _get_connection(conn_id)
+        except AirflowRuntimeError as e:
+            if e.error.error == ErrorType.CONNECTION_NOT_FOUND:
+                raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined") from None
+            raise
```

Comment:
We should just change this to use `airflow.sdk.Connection.get` to remove this code duplication.

Reply:
Handled in 534147bbe6